build.ps1

```diff
@@ -504,6 +550,9 @@ try
 		. $Nuget pack nuget\cef.redist.nuspec -NoPackageAnalysis -Version $CefPackageVersion -Properties 'Configuration=Release;Platform=x64;CPlatform=windows64;' -OutputDirectory nuget
 		. $Nuget pack nuget\chromiumembeddedframework.runtime.win.nuspec -NoPackageAnalysis -Version $CefPackageVersion -Properties 'Configuration=Release;Platform=x64;CPlatform=windows64;' -OutputDirectory nuget
 		
+		# Build arm64 packages
```

Comment:
Whilst it's unlikely we'll distribute a cef.redist Nuget package, we'll need one at least in the short term for the main project itself as both solutions rely on those packages.

Reply:
Thank you, I forgot about that. I added it as `cef.redist.arm64`.